src/lua/api-gateway/validation/oauth2/oauthTokenValidator.lua

```diff
@@ -78,6 +79,9 @@ end
 -- @param json Token info object
 --
 function _M:isCachedTokenValid(json)
+    if (json == nil) then
+        return -1
```

Comment:
why?

Reply:
Because the result is compared to zero further along.